### PR TITLE
Fix to ignore node_modules path in build

### PR DESCRIPTION
**Pull request by BrianFarnhill**
I recently came across a bug that was being causes by long file paths in the "node_modules" path when the AppVeyor builds were copying the module to the system modules directory - it would throw the PathTooLongException. To avoid this I've made a small update here to get it to ignore the node_modules folder as it shouldn't be needed here anyway.

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/dscresource.tests/134)

### CODE REVIEW

File: Meta.Tests.ps1 [hunk 1]
```diff
@@ -446,7 +446,11 @@ Describe 'Common Tests - Validate Example Files' -Tag 'Examples' {
         {
             $powershellModulePath = Join-Path -Path (Join-Path -Path $env:SystemRoot -ChildPath 'System32\WindowsPowerShell\v1.0\Modules') -ChildPath $repoName
 
-            Copy-item -Path $env:APPVEYOR_BUILD_FOLDER -Destination $powershellModulePath -Recurse -Force
+            Get-ChildItem $env:APPVEYOR_BUILD_FOLDER -Recurse | Where-Object -FilterScript {
```
Comment (PlagueHO) [inline Meta.Tests.ps1]: Can you convert the Get-ChildItem Path to a Named parameter?

Comment (BrianFarnhill) [inline Meta.Tests.ps1]: Done - sorry for the delay, I missed the notification on this!

**Comment by johlju** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[Meta.Tests.ps1, line 449 at r2](https://reviewable.io:443/reviews/powershell/dscresource.tests/134#-KgFVYHrnZJGEYHQULL5:-KgFVYHrnZJGEYHQULL6:b-exn6tv) ([raw file](https://github.com/powershell/dscresource.tests/blob/81c2b91922cf041a5718cf272bccc6bd6f6ed0b0/Meta.Tests.ps1#L449)):*
> ```PowerShell
>             $powershellModulePath = Join-Path -Path (Join-Path -Path $env:SystemRoot -ChildPath 'System32\WindowsPowerShell\v1.0\Modules') -ChildPath $repoName
> 
>             Get-ChildItem -Path $env:APPVEYOR_BUILD_FOLDER -Recurse | Where-Object -FilterScript {
> ```

I think this one could be considered extensive piping?
https://github.com/PowerShell/DscResources/blob/master/BestPractices.md#use-variables-rather-than-extensive-piping

Maybe add result from Get-ChildItem into a variable and use Copy-Item on that variable? 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/134)*

**Comment by BrianFarnhill** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[Meta.Tests.ps1, line 449 at r2](https://reviewable.io:443/reviews/powershell/dscresource.tests/134#-KgFVYHrnZJGEYHQULL5:-KgHZJPEUHBC_CZioUtD:bdhjzut) ([raw file](https://github.com/powershell/dscresource.tests/blob/81c2b91922cf041a5718cf272bccc6bd6f6ed0b0/Meta.Tests.ps1#L449)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

I think this one could be considered extensive piping?
https://github.com/PowerShell/DscResources/blob/master/BestPractices.md#use-variables-rather-than-extensive-piping

Maybe add result from Get-ChildItem into a variable and use Copy-Item on that variable? 
</blockquote></details>

I disagree that this makes it more readable, but the standard says no more than 1 pipe per line so that's what we'll go with I guess. Updated. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/134)*

**Comment by johlju** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/134#-:-KgUdh8YorQyRJAqIxpF:bnfp4nl)*

**Comment by johlju** [comment]
@PlagueHO You have a comment to acknowledge on this still it seems. 😄

**Comment by PlagueHO** [comment]
Doh - thanks for the reminder @johlju

:lgtm:

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/dscresource.tests/134#-:-KjMb5_74kDlcIWNcuYB:b-spft64)*

**Comment by johlju** [comment]
@mbreakey3 or @kwirkykat can you glance thru this and merge it whenever you have a chance?

**Comment by mbreakey3** [comment]
Looks good - can you update the README though?

**Comment by BrianFarnhill** [comment]
@mbreakey3 Done

**Pull request merged**